src/web/app/pages-help/instructor-help-page/instructor-help-panel/instructor-help-panel.component.ts

```diff
@@ -27,6 +31,26 @@ export class InstructorHelpPanelComponent {
     this.isPanelExpandedChange.emit(value);
   }
 
-  constructor(public elementRef: ElementRef) { }
+  constructor(public elementRef: ElementRef,
+              private router: Router,
+              private activatedRoute: ActivatedRoute,
+              private location: Location,
+              private clipboard: Clipboard,
+  ) { }
 
+  copyUrlToClipboard(event: Event): void {
```

Comment:
Based on the current implementation, this shouldn't be named `copyUrlToClipboard`. The function does more than just copying the link to clipboard.